common/src/thrift/block_master.thrift

```diff
@@ -17,46 +17,46 @@ service BlockMasterService {
 
   // Tachyon Client API
 
-  /*
+  /**
```

Comment:
Maybe we should do this in a separate PR along with adding all the other javadoc niceness.